utils/kvrocks2redis/sync.cc

```diff
@@ -105,14 +106,10 @@ Status Sync::auth() {
   if (!config_->kvrocks_auth.empty()) {
     const auto auth_command = Redis::MultiBulkString({"AUTH", config_->kvrocks_auth});
     auto s = Util::SockSend(sock_fd_, auth_command);
-    if (!s.IsOK()) return Status(Status::NotOK, "send auth command err:" + s.Msg());
-    std::string line;
-    s = Util::SockReadLine(sock_fd_, &line);
-    if (!s.IsOK()) {
-      return Status(Status::NotOK, std::string("read auth response err: ") + s.Msg());
-    }
+    if (!s.IsOK()) s.Prefixed("send auth command err");
```

Comment:
Perhaps you forgot the `return`  keyword before `s.Prefixed(...)`

Reply:
Thanks, I will fix it.

Reply:
Done.